creator-node/src/contentAccess/helpers.ts

```diff
@@ -0,0 +1,24 @@
+import { getRegisteredDiscoveryNodes } from '../utils/getRegisteredDiscoveryNodes'
```

Comment:
one implementation note for `getRegisteredDiscoveryNodes` - think we can actually just hold this in proc as well to remove the redis call in most cases?
would still want the 30min expiry

Reply:
Wanna talk about this after standup? I don't fully understand.

(This import is left over from the old `premiumContentMiddlware`)

Reply:
nah this is not blocking, just a perf optimization to remove redis call here, but we can circle back to this after other changes are addressed